Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:onbuild
+RUN mkdir /app
+ADD . /app/
+WORKDIR /app
+RUN go get
```

Comment:
I had to add the following line to be able to build the docker image:
`ENV GOBIN=$GOPATH/bin`